README.md

```diff
@@ -1,2 +1,27 @@
-# census-rm-deploy
-Response Management deployment pipelines and documentation 
+# Census RM Deploy
+Response management Concourse pipelines and documentation
+
+## Pipelines
+### CI Kubernetes Pipeline
+Specified in `pipelines/ci-kubernetes-pipeline.yml`, this pipeline has two functions.
+
+The first pulls from the `census-rm-kubernetes` repo triggered by commits to master and applies the latest microservice and handler configs to the kubernetes cluster specified by the pipeline configuration.
+
+The second checks for new builds for the `latest` tag for the RM container registry images. On being triggered by a new build, it runs through the applying the config from master, then runs `kubectl patch` for the newly built images deployment giving them a timestamp and image digest label. This causes kubernetes to re-pull the latest image and bring up new pods while we are using `imagePullPolicy: Always`.
+
+#### How to fly
+Create a secrets YAML file containing the required configuration:
+
+| Variable | Format | Description |
+| --- | --- | ---: |
+| kubernetes-server | https://\<host>:\<port> | The URL of the kubernetes server endpoint |
+| kubernetes-namespace | String | The kubernetes namespace for concourse to deploy to |
+| kubernetes-token | JWT String | A JWT access token. Must be for an account with the required roles in kubernetes |
```

Comment:
might be worth being a bit more specific and mention "service account" not just "account" in the description